Generate 4.16.0 release note

hangc0276 opened the pull request:
### Motivation
Generate 4.16.0 release notes

dlg99 commented on site3/website/src/pages/release-notes.md:
```diff
@@ -1,6 +1,388 @@
+* BP-60:Change PCBC limitStatsLogging default value to true [PR #3719](https://github.com/apache/bookkeeper/pull/3719)
+
+#### Features
+* [FEATURE] Added api/v1/bookie/cluster_info REST API [PR #3714](https://github.com/apache/bookkeeper/pull/3714)
+* [FEATURE] Added api/v1/bookie/cluster_info REST API [PR #3713](https://github.com/apache/bookkeeper/pull/3713)
+* [feature] Added api/v1/bookie/cluster_info REST API [PR #3710](https://github.com/apache/bookkeeper/pull/3710)
```
three times? two of these are cherry-picks to older branches

dlg99 commented on site3/website/src/pages/release-notes.md:
```diff
@@ -1,6 +1,388 @@
+* BP-58: Change the API for org.apache.bookkeeper.stats.Counter [PR #3501](https://github.com/apache/bookkeeper/pull/3501)
+* Feature: auto recover support repaired not adhering placement ledger [PR #3359](https://github.com/apache/bookkeeper/pull/3359)
+* BP-50: Add reuse journal file feature to support Intel PMem disk [PR #3194](https://github.com/apache/bookkeeper/pull/3194)
+* BP-47 (task1): Add native IO library support [PR #3189](https://github.com/apache/bookkeeper/pull/3189)
+* BP-47 (task4): Aligned native buffer wrapper [PR #3253](https://github.com/apache/bookkeeper/pull/3253)
+* BP-47 (task6): Direct I/O entrylogger support [PR #3263](https://github.com/apache/bookkeeper/pull/3263)
+* BP-47 (task5): Garbage collection support direct IO entrylogger [PR #3256](https://github.com/apache/bookkeeper/pull/3256)
+* BP-47 (task7): DbLedgerStorage add direct entry logger support [PR #3366](https://github.com/apache/bookkeeper/pull/3366)
```
Please summarize this as a single line/one feature, list PR numbers or the master issue number. Otherwise it is too noisy and unclear

dlg99 commented on site3/website/src/pages/release-notes.md:
```diff
@@ -1,6 +1,388 @@
+* add metric cbThreadPoolQueueSize [PR #3424](https://github.com/apache/bookkeeper/pull/3424)
+* Optimize concurrent collection's shrink logic [PR #3417](https://github.com/apache/bookkeeper/pull/3417)
+* Rename bookkeeper metrics to specify OpenMetrics [PR #3406](https://github.com/apache/bookkeeper/pull/3406)
+* stage 10: bugfix indexDirs for ConvertToInterleavedStorageCommand [PR #3409](https://github.com/apache/bookkeeper/pull/3409)
+* stage 9: bugfix indexDirs for LedgersIndexRebuildOp [PR #3403](https://github.com/apache/bookkeeper/pull/3403)
+* stage 8: bugfix indexDirs for LedgersIndexCheckOp [PR #3402](https://github.com/apache/bookkeeper/pull/3402)
+* stage 7: print indexDirs's info when start bookie [PR #3401](https://github.com/apache/bookkeeper/pull/3401)
+* stage 6: bugfix indexDirs for LocationsIndexRebuildOp [PR #3399](https://github.com/apache/bookkeeper/pull/3399)
+* stage 5: bugfix indexDirs for readLedgerIndexEntries [PR #3398](https://github.com/apache/bookkeeper/pull/3398)
+* Stage 3~4 : Add indexDirs in statsLogger, add indexDirs's Listener.  [PR #3397](https://github.com/apache/bookkeeper/pull/3397)
```
summarize

dlg99 reviewed (changes requested):
A few comments.

i don't think we have to list all the changes, just outlines + the major ones, and a link to get all the changes.
Otherwise release notes getting too noisy and the notes' value as of summary vs list of all changes drops significantly.

hangc0276 commented:
> A few comments.
> 
> i don't think we have to list all the changes, just outlines + the major ones, and a link to get all the changes. Otherwise release notes getting too noisy and the notes' value as of summary vs list of all changes drops significantly.

@dlg99 Thanks for your suggestions. I have addressed all the comments, please help take a look, thanks.

eolivelli reviewed (approved):
+1

I support @dlg99 suggestion to remove useless stuff

dlg99 reviewed (approved):
Looks good!
